src/inverseRobot.js

```diff
@@ -8,6 +8,22 @@
 
 function inverseRobot(robot) {
   // write code here
+  const keys = Object.values(robot);
+  const duplicateElements = keys.filter(
+    (item, index) => keys.indexOf(item) !== index
+  );
+
+  if (duplicateElements.length > 0) {
+    return null;
+  }
+
+  const inversedRobot = {};
+
+  Object.keys(robot).forEach((value, index) => {
+    inversedRobot[keys[index]] = value;
+  });
```

Comment:
you can do all previous logic with this logic and iterate only ones (but you will need to rewrite it into for loop)
you check here if key existed in inversedRobot and return `null`